week_1/project/week_1.py

```diff
@@ -1,6 +1,7 @@
 import csv
 from datetime import datetime
 from typing import List
+from operator import attrgetter
```

Comment:
I like this because you skip using a lambda so the code (IMHO) ultimately ends up more readable for humans.